sdk/src/store.rs

```diff
@@ -113,20 +116,89 @@ impl Store {
     ///
     /// In most cases, calling [`Store::new()`] is preferred.
     pub fn new_with_label(label: &str) -> Self {
-        Store {
+        let mut store = Store {
             claims_map: HashMap::new(),
             manifest_box_hash_cache: HashMap::new(),
             claims: Vec::new(),
             label: label.to_string(),
+            #[cfg(feature = "openssl_sign")]
+            trust_handler: Box::new(crate::openssl::OpenSSLTrustHandlerConfig::new()),
+            #[cfg(all(not(feature = "openssl_sign"), target_arch = "wasm32"))]
+            trust_handler: Box::new(crate::wasm::WebTrustHandlerConfig::new()),
+            #[cfg(all(not(feature = "openssl_sign"), not(target_arch = "wasm32")))]
+            trust_handler: Box::new(crate::trust_handler::TrustPassThrough::new()),
             provenance_path: None,
-        }
+        };
+
+        // load the trust handler settings, don't worry about status as these are checked during setting generation
+        let _ = get_settings_value::<Option<String>>("trust.trust_anchors").map(|ta_opt| {
```

Comment:
There has to be a better way to do this. This looks really odd.

Reply:
Do you want to pass it in?